packages/components/switch/src/switch.vue

```diff
@@ -120,7 +120,7 @@ const { inputId } = useFormItemInputId(props, {
 
 const switchSize = useSize()
 const switchDisabled = useDisabled(computed(() => props.loading))
-const isModelValue = ref(props.modelValue !== false)
+const isControlled = ref(props.value === false)
```

Comment:
**Question:** why do we change here? IMHO, we should not change this at all as you said, we need to restore this